api.js

```diff
@@ -56,7 +61,17 @@ Api.prototype._runFile = function (file) {
 		args.push('--sorted');
```

Comment:
@jokeyrhyme No, we use use `child_prcess.fork` which is different. We could probably use it in the CLI tests, though.